ssc/Pos/Ssc/Extra/Logic.hs

```diff
@@ -235,9 +235,6 @@ sscApplyBlocksFinish
     => SscGlobalState ssc -> m [SomeBatchOp]
 sscApplyBlocksFinish gs = do
     sscRunGlobalUpdate (put gs)
-    inAssertMode $
-        logDebug $
-        sformat ("After applying blocks SSC global state is:\n"%build) gs
```

Comment:
It's often such a huge text. Is there a way to have it go to the file log but not stdout?